cmd/nerdctl/build_test.go

```diff
@@ -204,17 +204,6 @@ COPY %s /`,
 	assert.Equal(t, string(data), testContent)
 }
 
-func createBuildContext(dockerfile string) (string, error) {
```

Comment:
Why remove this?

Reply:
I had moved it to [`image_testing_utils.go`](https://github.com/containerd/nerdctl/pull/1393/files#diff-41f6b435fd39f41851874b66da27db4ec63cfe4a5d7ac9a168bf4acdf0037b8fR29) since it is also called in a couple of non-build-related tests.